{{cookiecutter.project_dir}}/charts/{{cookiecutter.project_dir}}-ui/values.yaml

```diff
@@ -63,6 +63,7 @@ canary:
 
 service:
   name: {{ cookiecutter.project_dir }}-ui
+  baseName: {{ cookiecutter.project_dir }}
```

Comment:
Instead of `baseName`, I would recommend defining a variable at the top for `project` or `projectName` which can later be used for additional k8s labels.